deploy/task-cad-checks-secrets-pd.yaml

```diff
@@ -1,11 +1,16 @@
 ---
-# to generate this secret use the command
-# oc create secret generic cad-pd-token -n configuration-anomaly-detection --from-literal=CAD_PD_TOKEN=${PD_TOKEN} --dry-run -oyaml > task-cad-checks-secrets-pd.yaml
 apiVersion: v1
 kind: Secret
 metadata:
   name: cad-pd-token
   namespace: configuration-anomaly-detection
 type: Opaque
 stringData:
+  CAD_ESCALATION_POLICY: CHANGEME
+  CAD_PD_EMAIL: CHANGEME
+  CAD_PD_PW: CHANGEME
   CAD_PD_TOKEN: CHANGEME
+  CAD_PD_USERNAME: CHANGEME
+  CAD_SILENT_POLICY: CHANGEME
+  X_SECRET_TOKEN: CHANGEME
```

Comment:
Change the docs that refer to this secret with the value CHANGEME

Reply:
@georgettica sorry, I totally missed this comment :D. I will make a card out of this and fix it.

Reply:
:)